pkg/controller/kogitoapp/resource/requested_resources.go

```diff
@@ -31,9 +31,10 @@ type Context struct {
 }
 
 type builderChain struct {
-	Context   *Context
-	Resources *KogitoAppResources
-	Error     error
+	Context      *Context
+	Resources    *KogitoAppResources
```

Comment:
We still need this structure for something? I mean, the k8s resources slices is meant for this role, right?

Reply:
We still need `KogitoAppResources`, because the code can get the specific resource objects with it. I have removed the k8s resources slices, it can be generated with `KogitoAppResources`.